library/src/main/java/com/novoda/downloadmanager/lib/Cursors.java

```diff
@@ -0,0 +1,27 @@
+package com.novoda.downloadmanager.lib;
+
+import android.database.Cursor;
+
+final class Cursors {
+
+    private Cursors() {
+        // no instances
+    }
+
+    public static int getInt(Cursor c, String column) {
+        return c.getInt(columnIndexFor(c, column));
+    }
+
+    public static long getLong(Cursor c, String column) {
+        return c.getLong(columnIndexFor(c, column));
+    }
+
+    public static String getString(Cursor c, String column) {
```

Comment:
not sure why we have these static helper methods? I would get rid of this class, next thing someone will be mocking these methods with powermock! ew